starter-code/views/recipes.hbs

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```

Comment:
This part could have been added as a partial because you've repeated it on multiple pages